autoload/dracula.vim

```diff
@@ -54,4 +54,13 @@ func! dracula#should_abort(...)
     return 0
 endfunction
 
+" Use termguicolors instead of 256
+if (has('nvim'))
+  let $NVIM_TUI_ENABLE_TRUE_COLOR = 1
+endif
+
+if (has('termguicolors'))
```

Comment:
This is just wrong--vim supports the option, but not every terminal supports truecolor. It is an option we should not set for the user! They know better than we if this is what they want. We provide both GUI and terminal colors--the scheme is designed to work for both (though not perfectly as you point out).

Reply:
Agree. I don't believe we should be responsible for this.